tests/scripts/create_subsets_ghost.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Create minimal testdataset for GHOST reader
+
+Created on Fri Feb 26 09:17:09 2021
+
+@author: jonasg
+"""
+import os
+
+import matplotlib.pyplot as plt
+
+plt.close("all")
```

Comment:
is this reevant?

Reply:
I put that into a separate issue #604. A bit out of scope for now.